frontend/common/kubeflow-common-lib/projects/kubeflow/src/lib/namespace-select/namespace-select.component.html

```diff
@@ -3,11 +3,15 @@
   <mat-form-field>
     <mat-label i18n>Select Namespace</mat-label>
     <mat-select
-      [(ngModel)]="currNamespace"
       name="namespacesSelect"
+      [value]="currNamespace"
       (selectionChange)="namespaceChanged($event.value)"
       data-cy-namespace-selector-dropdown
     >
+      <mat-option data-cy-all-namespaces value="{all}">
```

Comment:
Do we want the "All namespaces" option? I dont know if our jupyter-apis is able to handle that, although maybe that is a change we will implement once we get there.

Reply:
We keep it, see how it does. and worse case remove it after